piksi_multi_cpp/include/piksi_multi_cpp/observations/udp_observation_sender.h

```diff
@@ -0,0 +1,25 @@
+#ifndef PIKSI_MULTI_CPP_OBSERVATIONS_UDP_OBSERVATIONS_SENDER_H_
+#define PIKSI_MULTI_CPP_OBSERVATIONS_UDP_OBSERVATIONS_SENDER_H_
+#include <libsbp/observation.h>
+#include <libsbp/sbp.h>
+#include <libsbp/system.h>
+#include <vector>
+
+namespace piksi_multi_cpp {
+class UDPObservationsSender {
```

Comment:
Should this class be a specialized callback handler such that we can hook it up to the base station receiver?

Reply:
yep, done through a pipes&filter sort of structure (see PR comment now)